src/main/java/ru/practicum/shareit/item/model/Item.java

```diff
@@ -1,21 +1,37 @@
 package ru.practicum.shareit.item.model;
 
-import lombok.Data;
-import ru.practicum.shareit.request.ItemRequest;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonManagedReference;
+import lombok.Getter;
+import lombok.Setter;
+import ru.practicum.shareit.item.comment.model.Comment;
 import ru.practicum.shareit.user.model.User;
 
+import javax.persistence.*;
 import javax.validation.constraints.NotBlank;
 import javax.validation.constraints.NotNull;
+import java.util.List;
 
-@Data
+@Getter
+@Setter
+@Entity(name = "items")
+@JsonIgnoreProperties({"hibernateLazyInitializer"})
```

Comment:
А это Вам для чего?

Reply:
Без этой аннотации ловлю эксепшн:
_Servlet.service() for servlet [dispatcherServlet] in context with path [] threw exception [Request processing failed; nested exception is org.springframework.http.converter.HttpMessageConversionException: Type definition error: [simple type, class org.hibernate.proxy.pojo.bytebuddy.ByteBuddyInterceptor]; nested exception is com.fasterxml.jackson.databind.exc.InvalidDefinitionException: No serializer found for class org.hibernate.proxy.pojo.bytebuddy.ByteBuddyInterceptor and no properties discovered to create BeanSerializer (to avoid exception, disable SerializationFeature.FAIL_ON_EMPTY_BEANS) (through reference chain: ru.practicum.shareit.booking.model.Booking["item"]->ru.practicum.shareit.item.model.Item$HibernateProxy$pQY8p3Z0["hibernateLazyInitializer"])] with root cause_

Reply:
Вероятно видел https://stackoverflow.com/questions/52656517/no-serializer-found-for-class-org-hibernate-proxy-pojo-bytebuddy-bytebuddyinterc
Хотя честно говоря не очень радует такая ситуация, но ок, а то мы тут погрязнем...